src/Components/OperationsGroup.jsx

```diff
@@ -37,6 +37,12 @@ export default function OperationsGroup({
   const selectedElement = useStore((state) => state.selectedElement)
   const colorMode = useContext(ColorModeContext)
 
+  const isCollaborationGroupVisible = useStore((state) => state.isCollaborationGroupVisible)
+  const isModelInteractionGroupVisible = useStore((state) => state.isModelInteractionGroupVisible)
+  const isSettingsVisible = useStore((state) => state.isSettingsVisible)
+
+  const isFirstDividerVisible = useStore((state) => state.getFirstDividerVisiblility)
```

Comment:
Dividers should be a part of the group, so we do not need to explicitly control them.

Reply:
Users/Third-parties don't have an explicit control over them. Their visibility is derived from the visibility of the surrounding groups, because if they belonged to the group you will have visual situations where one group is visible (and the divider as well if it belonged to the group) but the divider should be invisible in that case.